Daytime-Nighttime-Interactions/slack-channel-usage.rst

```diff
@@ -0,0 +1,192 @@
+
+- **Tier 2:** General discussion channels whose information and/or contents are not time critical.
+  The shift managers monitor these channels to relay any pertinent information between external support and the summit teams.
+
+- **Tier 3:** Channels not directly associated with current summit activities. 
+  These are channels for collaborative work, which make up the high majority of Slack channels.
+
+
+.. figure:: ./_static/Slack-channel-diagram.png
+    :name: Slack-channel-diagram
+
+    The expected level of participation by summit personnel to observation-related slack channels.
+
+
+Individual Channel Descriptions and Guidelines
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This section provides additional details regarding the use of each channel.
+Note that all channels require active monitoring by everyone to ensure the conversation stays inline with the channel's expected content.
+The shift managers are the primary disciplinarians but observers and others should police too!
+
+#summit-announce
+----------------
+This channel is used to announce activities that may impact other summit users.
+Announcements on this nor any other channel does **NOT** grant permission from personnel on the ground regarding whether or not it is safe to perform the test.
+
+Any summit activities and/or testing, including anything utilizing the network, requires **two** announcements: one when starting, the other when finished, to prevent people from stepping on one another.
+People who participate in testing or infrastructure support/development are expected to be actively monitoring this channel.
+
+Because work is being coordinated across multiple channels,this channel aims to restrict the need to make announcements multiple places and avoid the scenario of accidentally not informing all impacted parties.
+
+For items having substantial impact, scheduling should first be carried out as part of the `SUMMIT Jira calendar <https://jira.lsstcorp.org/secure/DoItBetterCalendar.jspa>`_ process.
+
+**Announcement Protocol:**
+
+- Announcements are required for all work that may affect the summit control network or its infrastructure.
+  This includes hardware testing, CSC updates/deployments, configuration changes, Nublado updates, invasive networking changes and any on-site activities being performed by summit personnel.
+- Utilize threads whenever possible to minimize cross-talk and/or interference between simultaneous discussions
+
+**Announcement Guidelines and Procedure:**
+
+- Make the announcement when ready to begin work.
+  Include an estimate of how long the systems will be in use.
+  Include the names of other channels where conversation on the work will take place (if appropriate).
+  List the CSC(s) you plan on using and a 1 or 2-sentence synopsis of what you are doing.
+
+- Announce when work is completed.
+  This should not be buried in a previous thread but instead should be a new message.
```

Comment:
We have mostly been using threads to announce when the work is completed. I think this works better than having a separate message saying the work was completed as it links start/finish to the same message. Maybe we should use the "Also send to ..." feature? but only if there are other announcements after the original one?